internal/rollup/rollup_bundle.bzl

```diff
@@ -147,7 +167,22 @@ def _run_tsc(ctx, input, output):
       arguments = [args]
   )
 
-def run_uglify(ctx, input, output, debug = False, comments = True, config_name = None, in_source_map = None):
+def _run_tsc_on_directory(ctx, input_dir, output_dir):
```

Comment:
wow, this is surprising, I guess we should discuss the design here so I can understand why we have to do this